packages/toolkit/src/view/resource/ResourceListPageMainView.tsx

```diff
@@ -2,7 +2,12 @@
 
 import * as React from "react";
 import { AddConnectorDialog } from "./AddConnectorDialog";
-import { GeneralPageProp, useEntity, useUserConnectors } from "../../lib";
+import {
+  GeneralAppPageProp,
+  useAppEntity,
+  useEntity,
+  useUserConnectors,
+} from "../../lib";
```

Comment:
@iamnamananand996 This unused variables cause the app to collapse, please make sure you double check your implementation before pushing for review